ansible/roles/test/tasks/qos_sai.yml

```
@@ -208,7 +208,7 @@
         - pkts_num_hdrm_partial={{qp.hdrm_pool_size.pkts_num_hdrm_partial}}
       when: minigraph_hwsku is defined and
             (minigraph_hwsku == 'Arista-7060CX-32S-C32' or minigraph_hwsku == 'Celestica-DX010-C32' or minigraph_hwsku == 'Arista-7260CX3-D108C8'
-             or minigraph_hwsku == 'Force10-S6100')
+             or minigraph_hwsku == 'Force10-S6100' or minigraph_hwsku == 'Arista-7260CX3-Q64')
```

```suggestion
             minigraph_hwsku in ['Force10-S6100', 'Arista-7260CX3-Q64']
```
It also works for the above one.